bundles/org.openhab.binding.gpio/README.md

```diff
@@ -22,12 +22,25 @@ sudo raspi-config
 
 -> Interfacing Options --> Remote GPIO --> YES --> OK --> Finish
 
+Note: if you are setting this up on a Raspberry Pi running openHabian, raspi-config is no longer available. However, you can still create the service config file manually:
```

Comment:
I would suggest to reword it a bit different. Because if openHabian would get raspi-config in the future this comment will be outdated and thus would need to be changed. So I've generalized it to systems without raspi-config.:
```suggestion
Note: if you are setting this up on a Raspberry Pi without `raspi-config` you can create the service config file manually:
```